webapp/src/ts/modals/tour/tour-select.component.html

```diff
@@ -0,0 +1,15 @@
+<mm-modal
+  id="'tour-select'"
+  [titleKey]="'tour.select'"
+  (onCancel)="close()"
+>
+  <div role="document" class="tour-select">
```

Comment:
Like above, I took a general approach to make that CSS class available in the modal , so other components inside modals can make use of it, despite now only been used by the tour modal.

Reply:
I'd rather we don't pollute the abstract class.

Reply:
The class is unlikely to ever be useful to other modals because it's related to visible tabs. I sure we're not going to add more modals in the near future that will make use of it. Let's try to keep things clean, please. Thanks!

Reply:
Sorry I don't agree, but I'll. In the master version the `hide-mobile-tab-labels` class was in the `body` tag, not in a inner `div` tag. The changes in the migration moved that CSS out of the body causing modal components to not inherit the class any more. There are other modals not using the class but we don't know for sure if it could be useful in the future, because it defines how icons are shown, for now only the one at the top and the same ones in the tour, but may be useful for other workflows with icons displayed if we want to maintain the same look & feel. Moreover, this is only used if a modal is created, and once the modal is destroyed, the objects listening the variable are destroyed.

Reply:
It defines how icons are shown depending on how many tabs the user can see in mobile view. 
If any other tab needs it, we can add it then. Until then, I'd prefer to keep the abstract class clean.